scripts/test/e2e-test.ps1

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+. "$PSScriptRoot\..\_common.ps1"
+
+# Restore and compile the test app
+dotnet restore "$RepoRoot\test\E2E" --runtime "osx.10.10-x64" --runtime "ubuntu.14.04-x64" --runtime "win7-x64"
```

Comment:
please add check for exit code in here and line below - we are using our own stuff to test ourselves so we need to be pretty careful about that :)